src/cppcache/src/TcrMessage.hpp

```diff
@@ -1114,10 +1116,10 @@ class TcrMessageHelper {
     } else if (!isObj) {
       // otherwise we're currently always expecting an object
       char exMsg[256];
-      std::snprintf(exMsg, 255,
-                    "TcrMessageHelper::readChunkPartHeader: "
-                    "%s: part is not object",
-                    methodName);
+      snprintf(exMsg, 255,
```

Comment:
If you had to remove std:: to make this work then it isn't working like you think it is. Let's figure out why and fix correctly,